### ISSUE TITLE
Reference subcomponent in OutgoingPropagation

### ISSUE THREAD

**Issue by juli1** [osate/ErrorModelV2#98]
Considering the following code:
component error behavior
		-- mapping of GPS producing GPS output to the GPS out propagation
		-- since it is a subcomponent single port connected to the GPS port, we could infer the mapping 
		propagations
		FTAStart1: all-[processing.location{ServiceOmission}]-> location{ServiceOmission};
end component;

The condition element related to the outgoing propagation cannot contain the subcomponent. Thus, it is not possible to process an instance model. Here, we can access the error propagation (location with ServiceOmission) but NOT the subcomponent. I can get the classifier but this is useless when analyzing the instance model.

The condition references the subcomponent when referencing a state in a composite error behavior. For example in
		composite error behavior
		states
		FTAStart2: [processing.FailStop]-> FailStop;
		end composite;

then, the conditionelement contains a qualified state that has a subcomponent. We can access it through conditionElement.getQualifiedState().getSubcomponent() (conditionelement being the element represented by processing.FailStop

So, to sum up, we would need to be able to reference the subcomponent when referencing the error propagation in a condition.

**Comment by reteprelief** [comment]
supported now.
